storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java

```diff
@@ -1708,11 +1765,11 @@ Create the blob with string (plain text) content.
          Start with a URL that identifies a snapshot of a blob in a container and includes a Shared Access Signature
          (SAS).
          */
-        URL u = new URL("https://myaccount.blob.core.windows.net/mycontainter/ReadMe.txt?" +
-                "snapshot=2011-03-09T01:42:34.9360000Z" +
-                "&sv=2015-02-21&sr=b&st=2111-01-09T01:42:34Z&se=2222-03-09T01:42:34Z&sp=rw" +
-                "&sip=168.1.5.60-168.1.5.70&spr=https,http&si=myIdentifier&ss=bf&srt=s" +
-                "&sig=92836758923659283652983562==");
+        URL u = new URL("https://myaccount.blob.core.windows.net/mycontainter/ReadMe.txt?"
+                + "snapshot=2011-03-09T01:42:34.9360000Z"
+                + "&sv=2015-02-21&sr=b&st=2111-01-09T01:42:34Z&se=2222-03-09T01:42:34Z&sp=rw"
```

Comment:
Same, StringBuilder might be a neat and efficient choice for constructing this string

Reply:
Same reason above.